src/main/groovy/com/synopsys/integration/detect/configuration/DetectConfigurationFactory.java

```diff
@@ -34,14 +35,15 @@
 import com.synopsys.integration.detect.util.filter.DetectToolFilter;
 import com.synopsys.integration.detect.workflow.bdio.BdioOptions;
 import com.synopsys.integration.detect.workflow.blackduck.BlackDuckPostOptions;
+import com.synopsys.integration.detect.workflow.blackduck.DetectProjectServiceOptions;
 import com.synopsys.integration.detect.workflow.file.AirGapOptions;
 import com.synopsys.integration.detect.workflow.file.DirectoryOptions;
-import com.synopsys.integration.detect.workflow.blackduck.DetectProjectServiceOptions;
 import com.synopsys.integration.detect.workflow.project.ProjectNameVersionOptions;
 import com.synopsys.integration.detector.evaluation.DetectorEvaluationOptions;
 import com.synopsys.integration.detector.finder.DetectorFinderOptions;
 import com.synopsys.integration.util.EnumUtils;
 
+// TODO: Create private method for accessing properties that assumes a PropertyAuthority of NONE.
```

Comment:
This TODO seems like a great idea. I assume you're avoiding overloading this PR, but (assuming it's pretty quick) this seems like a great thing to do this release.

Reply:
I agree, and making them static classes with a generic type ie: DETECT_DETECTOR_SEARCH_EXCLUSION_PATHS<String> would also be a step in the right direction.